modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/service/impl/ClientExtensionEntryLocalServiceImpl.java

```diff
@@ -107,6 +108,9 @@ public ClientExtensionEntry addClientExtensionEntry(
 
 		_addResources(clientExtensionEntry);
 
+		_cetManager.addOrUpdateCET(
+			clientExtensionEntry.getCompanyId(), clientExtensionEntry);
```

Comment:
This looks problematic to call addOrUpdate before the workflow has run. I think this would result in a CET being updated or available before it was approved. Is that right?

Reply:
You're right

Reply:
Can you update this, since this is a key part of the change we want to run by the CX team?